build.ts

```diff
@@ -67,8 +68,27 @@ try {
 
 	const isDebugBuild = argv.includes('--debug');
 	const buildType = isDebugBuild ? 'development' : 'production';
+	let platform: string = 'null' ;
+	let extension: string = 'null' ;
+	switch(process.platform) {
+		case 'darwin': {
+			platform = 'macos';
+			extension = '.app';
+		}
+		break;
+		case 'linux': {
+			platform = 'linux';
+			extension = '';
+		}
+		break;
+		case 'win32': {
+			platform = 'windows';
+			extension = '.exe' ;
+		}
+		break;
```

Comment:
Add a default case to this switch to abort the script and report unsupported platform, otherwise it will start producing a `null-x64/updater.null` build.